e2e_postgres_test.go

```diff
@@ -162,6 +163,48 @@ func TestPostgresHook(t *testing.T) {
 		}
 		t.Log(now)
 	}
+	pgxv5.RegisterDriver("cloudsql-postgres")
```

Comment:
Could we turn this into a table test?

https://semaphoreci.com/blog/table-driven-unit-tests-go